ui/DesignSystem/Component/WalletStatusIndicator.svelte

```diff
@@ -32,7 +32,7 @@
         </SidebarItem>
       </Tooltip>
     {:else}
-      <Tooltip value="Wallet · Connected">
+      <Tooltip value={`Wallet · Connected to ${wallet.environment}`}></Tooltip>
```

Comment:
```suggestion
      <Tooltip value={`Wallet · Connected to ${wallet.environment}`}>
```

Reply:
Yes it slipped through in the last commit.. I fixed it in the most recent commit.